tests/component.test.tsx

```diff
@@ -177,4 +250,12 @@ describe('Component', () => {
     const eventA = {target: {value: 'a', selectionEnd: 1}, persist};
     expect(a.simulate('change', eventA).state('value')).toEqual('12:34');
   });
+
+  test('should handle character deletion (returnEmptyString = true)', () => {
+    const eventC = {target: {value: '12:3', selectionEnd: 4}, persist};
+    expect(c.simulate('change', eventC).state('value')).toEqual('');
+
+    const eventD = {target: {value: '12:34:5', selectionEnd: 4}, persist};
+    expect(d.simulate('change', eventD).state('value')).toEqual('');
```

Comment:
As I get it, this test describes the whole point to have `returnEmptyString` parameter. Please, correct me if I am wrong, the original intention is to clean up the time field just by pressing `DELETE` key once?

Reply:
Yes that's it, when the user press `DELETE` key or `BACKSPACE`